### DISCUSSION TITLE
Use of [prefix] in titles of issues and PRs

### DISCUSSION THREAD

**Discussion by fbrausse** [esbmc/esbmc#1098] [Polls]
Personally, I find these varying widths prefixes in brackets hard to read on the Github page, both for issues and for PRs. In my view they give additional context to the actual problem, but they are at the wrong place: at the start of the title. I would therefore like your opinion on avoiding those [prefixes] by replacing them with corresponding (colorful) Github labels, which are displayed at the end instead of the beginning.

**Comment by lucasccordeiro** [comment]
Switch to labels ;-)

**Comment by lucasccordeiro** [comment]
`Switch to labels` option has won here :-)
